mix.exs

```diff
@@ -35,8 +35,8 @@ defmodule StripJs.Mixfile do
     [
       {:floki, github: "philss/floki", ref: "af5dcdb"},
```

Comment:
can we see if Floki ever cut a new release so we can pin to that instead of that git hash? IIRC I pinned to that hash because they had not yet done so